_components/compatibility.md

```diff
@@ -76,134 +76,7 @@ If there is JavaScript to execute regardless of the experience served to the bro
 
 ## Browser support
 
-<aside>The <a href="https://docs.google.com/document/d/1mByh6sT8zI4XRyPKqWVsC2jUfXHZvhshS5SlHErWjXU/" class="o-typography-link--external">formal resources</a> for the <abbr title="Financial Times">FT</abbr>'s browser support are available to <abbr title="Financial Times">FT</abbr> staff.</aside>
-
-Origami components conform to the <abbr title="Financial Times">FT</abbr>'s browser support standard, which looks like this:
-
-
-##### DESKTOP
-<table class="o-table o-table--horizontal-lines o-layout__main__single-span" data-o-component="o-table">
-	<thead>
-		<tr>
-			<th>Browser</th>
-			<th>Enhanced</th>
-			<th>Core</th>
-		</tr>
-	</thead>
-	<tbody>
-		<tr>
-			<td>Chrome</td>
-			<td>Latest</td>
-			<td>—</td>
-		</tr>
-		<tr>
-			<td>Chromium</td>
-			<td>—</td>
-			<td>Latest</td>
-		</tr>
-		<tr>
-			<td>Edge</td>
-			<td>Latest</td>
-			<td>–</td>
-		</tr>
-		<tr>
-			<td>Firefox</td>
-			<td>Latest</td>
-			<td>–</td>
-		</tr>
-		<tr>
-			<td><abbr title="Internet Explorer">IE</abbr></td>
-			<td>11</td>
-			<td>9</td>
-		</tr>
-		<tr>
-			<td>Opera</td>
-			<td>—</td>
-			<td>Latest</td>
-		</tr>
-		<tr>
-			<td>Safari</td>
-			<td>10 (6.1)</td>
-			<td>6</td>
-		</tr>
-	</tbody>
-</table>
-
-##### MOBILE
-<table class="o-table o-table--horizontal-lines o-layout__main__single-span" data-o-component="o-table">
-	<thead>
-		<tr>
-			<th>Browser</th>
-			<th>Enhanced</th>
-			<th>Core</th>
-		</tr>
-	</thead>
-	<tbody>
-		<tr>
-			<td>Android Native</td>
-			<td>—</td>
-			<td>4</td>
-		</tr>
-		<tr>
-			<td>BlackBerry Native</td>
-			<td>–</td>
-			<td>10</td>
-		</tr>
-		<tr>
-			<td>Chrome</td>
-			<td>Latest</td>
-			<td>—</td>
-		</tr>
-		<tr>
-			<td>Firefox</td>
-			<td>—</td>
-			<td>Latest</td>
-		</tr>
-		<tr>
-			<td>Safari</td>
-			<td>9</td>
-			<td>4</td>
-		</tr>
-		<tr>
-			<td>Samsung Native</td>
-			<td>5 (3)</td>
-			<td>—</td>
-		</tr>
-	</tbody>
-</table>
-
-##### INTERNAL PRODUCTS
-<table class="o-table o-table--horizontal-lines o-layout__main__single-span" data-o-component="o-table">
-	<thead>
-		<tr>
-			<th>Browser</th>
-			<th>Enhanced</th>
-			<th>Core</th>
-		</tr>
-	</thead>
-	<tbody>
-		<tr>
-			<td>Chrome</td>
-			<td>Latest</td>
-			<td>—</td>
-		</tr>
-		<tr>
-			<td>Firefox</td>
-			<td>Latest</td>
-			<td>—</td>
-		</tr>
-		<tr>
-			<td><abbr title="Internet Explorer">IE</abbr></td>
-			<td>11</td>
-			<td>—</td>
-		</tr>
-		<tr>
-			<td>Safari</td>
-			<td>Latest</td>
-			<td>—</td>
-		</tr>
-	</tbody>
-</table>
+Origami follow the <a href="https://docs.google.com/document/d/1z6kecy_o9qHYIznTmqQ-IJqre72jhfd0nVa4JMsS7Q4/" class="o-typography-link--external"><abbr title="Financial Times">FT</abbr> Browser Support Policy document</a> available to <abbr title="Financial Times">FT</abbr> staff.
```

Comment:
Is this document the official browser support document? How do we get notified when it's content is changed? Is that possible?

Reply:
It's at least canonical (recent update, the old FT support document links there, has more detail than what's publicly listed on ft.com). I don't think there's a process for updating it regularly so am not confident in getting an update by a person -- unless there's a Google Docs feature

Reply:
Why is this not the official browser support document? https://help.ft.com/help/tools-services/browser-compatibility/what-browsers-does-the-ft-support/

Reply:
>Our publicly stated support policy is available on help.ft.com. It is aligned with this document but has less detail.

It also doesn't mention specialist titles or internal products